arcade/paths.py

```diff
@@ -266,19 +341,23 @@ def astar_calculate_path(start_point: Point,
 def has_line_of_sight(point_1: Point,
                       point_2: Point,
                       walls: SpriteList,
-                      max_distance: int = -1,
+                      max_distance: float = float("inf"),
                       check_resolution: int = 2) -> bool:
     """
-    Determine if we have line of sight between two points. Try to make sure
-    that spatial hashing is enabled on the wall SpriteList or this will be
-    very slow.
-
-    :param Point point_1: Start position
-    :param Point point_2: End position position
-    :param SpriteList walls: List of all blocking sprites
-    :param int max_distance: Max distance point 1 can see
-    :param int check_resolution: Check every x pixels for a sprite. Trade-off
-                                 between accuracy and speed.
+    Determine if we have line of sight between two points.
+
+    NOTE: Try to make sure that spatial hashing is enabled
+    on the wall SpriteList or this will be very slow.
```

Comment:
A warning directive may render better.
```suggestion
    .. warning:: Try to make sure spatial hashing is enabled on ``walls``!

                 If spatial hashing is not enabled, this function may run
                 very slowly!
 ```